tests/Nexucis/Tests/Elasticsearch/Helper/Nodowntime/IndexActionTest.php

```diff
@@ -2,8 +2,13 @@
 
 namespace Nexucis\Tests\Elasticsearch\Helper\Nodowntime;
 
+use Nexucis\Elasticsearch\Helper\Nodowntime\Exceptions\IndexAlreadyExistException;
+use Nexucis\Elasticsearch\Helper\Nodowntime\Exceptions\IndexNotFoundException;
+use Yoast\PHPUnitPolyfills\Polyfills\AssertionRenames;
+
 class IndexActionTest extends AbstractIndexHelperTest
 {
+    use AssertionRenames;
```

Comment:
just for my knowledge, what does it do or why is it needed ?

Reply:
To support old phpunit for php 7.1 and 7.2